docs/usage/getting-started/private-packages.md

```diff
@@ -371,8 +371,8 @@ The solution to this is that you should break your presets into public and priva
 
 ### Encrypting secrets
 
-It is strongly recommended that you don't commit secrets to repositories, including private ones, and this includes secrets needed by Renovate to access private modules.
-Therefore the preferred approach to secrets is that the bot administrator configures them as `hostRules` which are then applied to all repositories which the bot accesses.
+It is strongly recommended that you avoid committing secrets to repositories, including private ones, and this includes secrets needed by Renovate to access private modules.
```

Comment:
We should put our point positively: [^quote]

> ## Use positive language
> 
> We’re accustomed to thinking and speaking positively. When we write in the negative, we place another stumbling block in audience’s way and make it more difficult for them to understand us. When you’re going to meet a friend at the airport, do you say, “If you fail to arrive by 5:00, I cannot pick you up,” or do you say, “You have to arrive by 5:00 if you want me to pick you up?”

[^quote]: [Plain language guidelines, use positive language](https://www.plainlanguage.gov/guidelines/concise/use-positive-language/)